src/lib/listening_groups.py

```diff
@@ -26,34 +24,29 @@ def tag_listening_groups_participants(cls, user, data, pipeline_configuration, r
         :param pipeline_configuration: Pipeline configuration.
         :type pipeline_configuration: PipelineConfiguration
         """
-        listening_group_participants = dict()   # Contains lists of weekly listening group participants. The participants
-                                                # will change each week.
-
-        # Read weekly listening group participants CSVs and add their uids to the respective radio-show
-        # listening_group_participants lists
-        listening_group_csvs = []
-        for listening_group_csv_url in pipeline_configuration.listening_group_csv_urls:
-            listening_group_csvs.append(listening_group_csv_url.split("/")[-1])
-        for plan in PipelineConfiguration.RQA_CODING_PLANS:
-            listening_group_participants[plan.dataset_name] = set()
-            if plan.listening_group_filename in listening_group_csvs:
-                with open(f'{raw_data_dir}/{plan.listening_group_filename}', "r",
-                          encoding='utf-8-sig') as f:
-                    plan_listening_group_data = list(csv.DictReader(f))
-                    for row in plan_listening_group_data:
-                        listening_group_participants[plan.dataset_name].add(row['avf-phone-uuid'])
-                    log.info(f'Loaded {len(listening_group_participants[f"{plan.dataset_name}"])} '
-                             f'{plan.dataset_name} listening group participants')
-            else:
-                log.warning(f'Skipping loading {plan.listening_group_filename},file not found!')
-
-        # 1.Check if a participant is part of the repeat listening groups contacts then tag true or false otherwise
-        #   Example - "repeat_listening_group_participant": true
-        # 2.Check if a participant participated in a radio show listening group then tag true or false otherwise
-        #   Example - "kakuma_s01e01_listening_group_participant": false
+
+        listening_group_participants = {
+            "health_practitioners": set(),
+            "mothers":set()
+        }
+
+        # Read listening group participants CSVs and add their uids to the respective group
+        for k in listening_group_participants.keys():
+            for listening_group_csv_url in pipeline_configuration.listening_group_csv_urls[f'{k}']:
```

Comment:
I don't think we need to use the f-string here. The dict keys are strings.

Reply:
Thanks for cathing this. Have removed them.